### PR TITLE
Upgrade gems

### PR DESCRIPTION
**Pull request by klssmith**
This upgrades most of the gems in the Gemfile in preparation for upgrading to Rails 5. 
(web-console and kaminari cannot be upgraded before the Rails upgrade.) 

[Trello card](https://trello.com/c/UvadVLm8/72-upgrade-dependencies-on-our-applications-including-rails)

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -42,7 +41,7 @@ group :development do
 end
 
 group :test do
-  gem 'webmock', '~> 1.22'
+  gem 'webmock', '~> 3.0'
```
Comment (whoojemaflip) [inline Gemfile]: is there value in pinning the versions of `test` gems?

Comment (klssmith) [inline Gemfile]: I don't know - I can remove the version here.

Comment (whoojemaflip) [inline Gemfile]: :+1:

Comment (klssmith) [inline Gemfile]: @whoojemaflip Now fixed.

**Pull request merged**